obspy/core/inventory/inventory.py

```diff
@@ -619,7 +650,12 @@ def select(self, network=None, station=None, location=None, channel=None,
             net_ = net.select(
                 station=station, location=location, channel=channel, time=time,
                 starttime=starttime, endtime=endtime,
-                sampling_rate=sampling_rate, keep_empty=keep_empty)
+                sampling_rate=sampling_rate,
+                minlatitude=minlatitude, maxlatitude=maxlatitude,
+                minlongitude=minlongitude, maxlongitude=maxlongitude,
+                latitude=latitude, longitude=longitude,
+                minradius=minradius, maxradius=maxradius,
+                keep_empty=keep_empty)
```

Comment:
Please move `keep_empty` to old position in kwargs list. It's very unlikely that people specify everything as args, but it still has potential to break codes.

Reply:
I thought about that: it seems odd to me to have this `keep_empty` argument right in the middle of the arglist.

But if that's what you prefer, I'll do it 😉

Reply:
yes, please.. potential code breaks trump cosmetics :smiley:

Reply:
Done. Will squash and rebase before merging.